build/ci/net8/global.json

```diff
@@ -0,0 +1,14 @@
+{
+	"sdk": {
+		"version": "8.0.100",
+		"allowPrerelease": true,
+		"rollForward": "latestPatch"
```

Comment:
Do we want to set this to "disable" for CI stability?

Reply:
Yes, let's freeze.